www/templates/intro/survey.html

```diff
@@ -1,5 +1,5 @@
 <ion-content class="has-footer">
-    <tutorial/>
+    <tutorial isValid="surveyState.isValid" result="surveyState.result" schema="surveyState.schema"/>
```

Comment:
@shankari I have confirmed your skepticism that this does not work. We are still not connected back to the angular code. I am working to figure out how this can be done.